CHANGELOG.md

```diff
@@ -1,4 +1,24 @@
-## 0.7.1 (Unreleased)
+## 0.8.1 (Unreleased)
+
+## 0.8.0 (Feb 11, 2019)
```

Comment:
This line is not needed.
The release tooling will insert the timestamp in place of the `Unreleased` word on the first line.